### PR TITLE
Add icons to build script, fix script and icons page design

### LINKED ISSUE #977: New docs design improvements
Just some small possible design improvements of the beautiful new docs :sparkles: 

- [x] Logo takes a lot of space in the navigation, could be a bit smaller. #986
- [x] On hover/focus of the logo, it should change opacity instead of getting this background box. #986
- [ ] Is it possible to use Nunito as font? See files at https://github.com/nextcloud/server/tree/master/core/fonts
- [ ] Adjust button styles to be round like in Nextcloud, primary button in #0082c9
- [ ] There are 2 "Icons" entries in design guidelines
- [ ] The icons page has list layout, should be grid: https://docs.nextcloud.com/server/15/developer_manual/design/icons.html

### PR DESCRIPTION
**Pull request by skjnldsv**
Closes #977 

@nextcloud/designers 

capture d ecran_2018-12-07_14-56-40

### CODE REVIEW

**Comment by skjnldsv** [comment]
All clear! please review! :)

File: Makefile [hunk 1]
```diff
@@ -36,13 +36,21 @@ user-manual-de-pdf:
 	@echo "User manual de build finished; PDF is updated"
 
 api-docs: clean-api-docs
-	cd build && sh get-server-sources.sh master
+	cd build && sh get-server-sources.sh $(DRONE_BRANCH)
```
Comment (MorrisJobke) [inline Makefile]: As we are not building this on drone for the live system we need to add the env variable there as well. Let me do that.

Comment (skjnldsv) [reply Makefile]: Okay :)
Thanks a lot Morris!

**Comment by MorrisJobke** [comment]
bildschirmfoto 2018-12-07 um 16 42 04

**Comment by skjnldsv** [comment]
@MorrisJobke Hum, did you installed the npm package for that?
did it not failed hard when building?

**Comment by MorrisJobke** [comment]
> @MorrisJobke Hum, did you installed the npm package for that?
> did it not failed hard when building?

I installed npm and make just finished it seems.

**Comment by skjnldsv** [comment]
@MorrisJobke then I need to fail harder if the package is not here I guess

**Comment by MorrisJobke** [comment]
Works now. 👍

**Comment by MorrisJobke** [comment]
> @MorrisJobke then I need to fail harder if the package is not here I guess

Correct - package was missing.

**Pull request merged**